src/core/LayerZeroOFT.sol

```diff
@@ -7,9 +7,18 @@ import {SafeTransferLib} from "../vendor/SafeTransferLib.sol";
 contract LayerZeroOFT {
     using SafeTransferLib for IERC20;
 
-    function bridgeLayerZeroOFT(IERC20 token, address oft, bytes memory sendData) internal {
-        uint256 updatedInputAmount = token.fastBalanceOf(address(this));
-        token.safeApproveIfBelow(oft, updatedInputAmount);
+    IERC20 internal constant ETH = IERC20(0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE);
```

Comment:
Somewhat idle thought: we should enshrine https://eips.ethereum.org/EIPS/eip-7528 somewhere in this repo rather than redefining this ad-hoc wherever we need it